test/e2e/search_flow.go

```
@@ -18,12 +41,14 @@ func testSearchFlow(t *T) error {
 		return err
 	}
 
+	// set go as the language in the browser
 	selectLang := t.FindElement(selenium.ById, "e2etest-search-lang-select-golang")
 	selectLang.Click()
-	searchInput := t.WaitForElement(selenium.ById, "e2etest-search-input")
-	searchInput.SendKeys("RouteMatch")
 
-	t.WaitForElement(selenium.ByTagName, "a", MatchAttribute("href", `/github\.com/gorilla/mux(@[^/]+)?/-/info/GoPackage/github.com/gorilla/mux/-/RouteMatch`))
+	to_test := [5]string{"new http request", "read file", "json encoder", "sql query", "indent json"}
+	for _, element := range to_test {
+		runSearchFlow(t, element)
```

you are ignoring the error returned